content/docs/user-guide/testing/getting-started/_index.md

```diff
@@ -0,0 +1,233 @@
+
+In short, all tests in O3DE use CTest as a high level test coordinator. When executed, CTest calls other lower-level test tools and then reports on their success.
+
+![CTest calls other test runners](/images/user-guide/testing/getting-started/ctest_to_runners.png)
+
+### Starting CTest
+
+CTest expects its working directory to be a CMake build directory, so be sure to first navigate to this directory:
+
+```shell
+cd <local_path_to>/o3de/build/<build_folder>
+ctest -C <build_configuration>
+```
+
+You may not want to run the commands above without also filtering the set of tests to run. Without a filter the command will run every test, and likely result in multiple hours of test execution! If you ever want to stop CTest, send an interrupt signal by selecting the terminal and pressing `CTRL + C` . The most commonly used suites are Smoke and Main, which have requirements to execute fast. The following ctest command filters to only tests in these two suites:
+
+```shell
+cd <local_path_to>/o3de/build/<build_folder>
+ctest -C <build_configuration> -L "(SUITE_smoke|SUITE_main)"
+```
+
+The build configuration should be one you have already built locally. Running the Smoke and Main suites for the `profile` build configuration on a Linux machine should look like:
+
+```shell
+cd user/github/o3de/build/linux
+ctest -C profile -L "(SUITE_smoke|SUITE_main)"
+```
+
+After running CTest, results save to `.../<build_folder>/Testing/`.  If you prefer to see a summary of failures in your terminal, add the flag `--output-on-failure`.
+
+For more information on CTest usage, refer to its [online documentation](https://cmake.org/cmake/help/latest/manual/ctest.1.html).
+
+### Adding test modules to CTest
+
+CTest registers entire modules of test code. These modules typically contain multiple individual tests of the same feature. Adding new tests has three major steps, with a prerequisite:
+
+0. Add a build target for the production code which tests will target.
+1. Add a build target for tests which need to be compiled (not required for Python).
+2. Register your test module in a `CMakeLists.txt`.
+3. Add individual test functions into the test module.
+
+Specific steps vary for different types of tests, discussed in later sections.
+
+## GoogleTest
+
+Many tests are easiest to write in the same language as the production code they target. Since the majority of O3DE uses C++, the majority of its tests also use C++. O3DE's C++ tests are typically small [unit level](https://softwaretestingfundamentals.com/unit-testing/) tests of specific low-level functionality, which run extremely fast. O3DE uses [GoogleTest](https://github.com/google/googletest/blob/main/docs/index.md) to coordinate these tests, with an execution wrapper named AzTestRunner. CTest will call AzTestRunner to load the test library and the target library, and then execute any loaded GoogleTest tests.
```

Comment:
Suggestions to make sentences flow better and fix style syntax.
```suggestion
For many tests, it's easier to write them in the same language as the production code that they target. Since the majority of O3DE uses C++, the majority of its tests also use C++. O3DE's C++ tests are typically small [unit level](https://softwaretestingfundamentals.com/unit-testing/) tests of specific low-level functionality, which run extremely fast. O3DE uses [GoogleTest](https://github.com/google/googletest/blob/main/docs/index.md) to coordinate these tests, with an execution wrapper named `AzTestRunner`. CTest calls `AzTestRunner` to load the test library and the target library, and then execute any loaded GoogleTest tests.
```